tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -128,6 +128,36 @@ const shareFileFolder = function (
       httpHelper.checkOCSStatus(json, 'Could not create share. Message: ' + json.ocs.meta.message)
     })
 }
+/**
+ * create any share using dataTable
+ *
+ * @param {string} sharer
+ * @param {object} dataTable (attrs like: path, shareWith, expireDate, name, permissionString,
+ * password can be passed inside dataTable)
+ *
+ * @return void
+ */
+Given('user {string} has created a new share with following settings',
+  function (sharer, dataTable) {
+    const settings = dataTable.rowsHash()
+    let expireDate = settings.expireDate
+    if (typeof settings.expireDate !== 'undefined') {
```

Comment:
here you can use `expireDate` instead of `settings.expireDate`

Reply:
done! :rocket: